codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateEncoderTest.java

```diff
@@ -96,6 +96,34 @@ public void testAlreadyCompressedFrame() {
     }
 
     @Test
+    public void testUncompressedWebSocketFrame() {
+        EmbeddedChannel encoderChannel = new EmbeddedChannel(new PerMessageDeflateEncoder(9, 15, false));
+
+        // initialize
+        byte[] payload = new byte[300];
+        random.nextBytes(payload);
+
+        UncompressedWebSocketFrame frame = new UncompressedWebSocketFrame(
+                new BinaryWebSocketFrame(true, WebSocketExtension.RSV3, Unpooled.wrappedBuffer(payload)));
+
+        // execute
+        encoderChannel.writeOutbound(frame);
+        BinaryWebSocketFrame newFrame = encoderChannel.readOutbound();
+
+        // test
+        assertNotNull(newFrame);
+        assertNotNull(newFrame.content());
+        assertTrue(newFrame instanceof BinaryWebSocketFrame);
```

Comment:
@sunng87 no need to assert this as if this not true it would have caused a `ClassCastException` before.